tests/test_parser/test_complex_outputs.xml

```diff
@@ -130,7 +130,7 @@
                 <container classes="cell_output" nb_element="cell_code_output">
                     <container nb_element="mime_bundle">
                         <container mime_type="image/png">
-                            <image candidates="{'*': '_build/jupyter_execute/16832f45917c1c9862c50f0948f64a498402d6ccde1f3a291da17f240797b160.png'}" uri="_build/jupyter_execute/16832f45917c1c9862c50f0948f64a498402d6ccde1f3a291da17f240797b160.png">
+                            <image candidates="{'*': '_build/jupyter_execute/16832f45917c1c9862c50f0948f64a498402d6ccde1f3a291da17f240797b160.png'}" height="400" uri="_build/jupyter_execute/16832f45917c1c9862c50f0948f64a498402d6ccde1f3a291da17f240797b160.png">
```

Comment:
This image should not get `height=400` it has no explicit metadata about this, and if matplotlib does provide any metadata on this it will be on both height and width, not only one of them.

Here is how the image looks before this PR:

![imatge](https://github.com/executablebooks/MyST-NB/assets/23738400/48fb7cdd-be6d-4a36-958b-2a0d01dc204b)

and after this PR:

![imatge](https://github.com/executablebooks/MyST-NB/assets/23738400/30ed6565-2934-4194-8b6f-2f591b2fe66e)

as it is forced to have height=400, the image is enlarged over its resolution.

Reply:
You’re right, the text/plain repr is `"<Figure size 432x288 with 1 Axes>"` so this should have metadata that represents that or none.